client/src/components/atoms/BannerCarousel.tsx

```diff
@@ -0,0 +1,27 @@
+import { Carousel, Container } from "react-bootstrap";
+import React, { FunctionComponent } from "react";
+import DonorHomepageConfig from "../../config/donorHompageConfig.json";
+
+const BannerCarousel: FunctionComponent = () => {
+    const bannerImgs = DonorHomepageConfig.Banner;
+    const bannerIntv = 1000 * 10;
+    return (
+        <div className="homepage-banner">
```

Comment:
Change `className` to match the name of the component. In the future, if someone creates a component called `HomepageBanner`, then they can use the css class `homepage-banner`
```
<div className="banner-carousel">
```